static/templates/user-info-form-modal.handlebars

```diff
@@ -15,6 +15,18 @@
                 <div class="input-group edit_bot_owner_container">
                     <label for="bot_owner_select">{{t "Owner" }}</label>
                 </div>
+                {{else if is_admin}}
+                <div class="user-admin-settings">
+                    <button class="button rounded small remove-admin btn-danger">
+                        {{t "Remove admin" }}
+                    </button>
+                </div>
+                {{else}}
+                <div class="user-admin-settings">
+                    <button class="button rounded small make-admin btn-warning">
+                        {{t "Make admin" }}
+                    </button>
+                </div>
```

Comment:
This is fine for now, but an upcoming step should be to make it a dropdown.

Reply:
(If we do the dropdown migration, the labels can be e.g.

"**Role**
[Guest]
[Member]
[Organization administrator]
")